setup.py

```diff
@@ -4,24 +4,30 @@
 
 from setuptools import setup
 
-setup(name='codebasin',
-      version='1.1.0',
-      description='Code Base Investigator',
-      author='John Pennycook',
-      author_email='john.pennycook@intel.com',
-      url='https://www.github.com/intel/code-base-investigator',
-      packages=['codebasin', 'codebasin.walkers'],
-      scripts=['codebasin.py'],
-      classifiers=['Development Status :: 3 - Alpha',
-                   'Environment :: Console',
-                   'Intended Audience :: Developers',
-                   'License :: OSI Approved :: BSD License',
-                   'Programming Language :: Python',
-                   'Topic :: Software Development'],
-      python_requires='>=3.9',
-      install_requires=['numpy',
-                        'matplotlib',
-                        'pyyaml',
-                        'scipy>=1.11.1',
-                        'jsonschema']
-      )
+setup(
+    name="codebasin",
+    version="1.1.0",
```

Comment:
Do you need to do a patch increment?

Reply:
Honestly, I'm not sure.  Strictly speaking, I don't think we need to increment the number until we settle on something that we're going to release... but that might make it hard to determine when we should release 1.1.z.

We could start incrementing the number in these PRs to help us keep track of how many bugs we've fixed, even if we don't actually release those versions.  What do you think?

Reply:
I think we should increment it just to help track as you say. Especially if the PR has "bug" fix in the title